userspace/libsinsp/metrics_collector.h

```diff
@@ -339,7 +339,7 @@ class libs_metrics_collector
 private:
 	sinsp* m_inspector;
 	std::shared_ptr<sinsp_stats_v2> m_sinsp_stats_v2;
-	uint32_t m_metrics_flags = METRICS_V2_KERNEL_COUNTERS | METRICS_V2_LIBBPF_STATS | METRICS_V2_RESOURCE_UTILIZATION | METRICS_V2_STATE_COUNTERS | METRICS_V2_PLUGINS;
+	uint32_t m_metrics_flags = METRICS_V2_KERNEL_COUNTERS | METRICS_V2_LIBBPF_STATS | METRICS_V2_RESOURCE_UTILIZATION | METRICS_V2_STATE_COUNTERS | METRICS_V2_PLUGINS | METRICS_V2_KERNEL_COUNTERS_PER_CPU;
```

Comment:
do we want to have it enabled by default here?

Reply:
I'd say yes because we always went with default value => enable everything.
@incertum wdyt?

Reply:
Looking at this comment https://github.com/falcosecurity/libs/issues/2028#issuecomment-2315830346 i would say that we want it disabled by default

Reply:
Oh but enabled by default here does not mean enabled by default in Falco, right? Anyway, it works fine either way for me

Reply:
yep probably we can just disable it in Falco by default